features/account/customer_account/viewing_orders_history/viewing_order_recalculated_by_exchange_rate.feature

```diff
@@ -11,7 +11,7 @@ Feature: Viewing details of an order
         And the store operates on a channel named "Web"
         And that channel allows to shop using the "USD" currency
         And that channel allows to shop using the "GBP" currency with exchange rate 3.0
-        And that channel uses the "USD" currency by default
+        And that channel uses the "USD" currency as base
```

Comment:
Not exactly. There is a [concept](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Core/Model/Channel.php#L29) of ``baseCurrency`` on ``Channel`` entity and it will not be deleted in the nearest (or further) future. So the sentence ``And that channel's base currency is "USD"`` is 100% correct now and in the future. ``And that channel uses the "USD" currency by default`` is not 100% correct neither now nor in the future, IMO the same with ``And that channel operates on "USD" currency``. ``channel's base currency`` is not open to interpretation, it's exactly what we do with channel (especially as it's the setup step).